.github/workflows/build.yml

```diff
@@ -29,6 +29,12 @@ jobs:
         with:
           redis-version: '6.2'
           auto-start: "false"
+      
+      - name: "Install TiUP"
```

Comment:
I assume this has to be installed locally for dev/test as well, right?

Reply:
Yes, this is similar to the redis cli